cypress/integration/create-recipe.spec.js

```diff
@@ -0,0 +1,11 @@
+/// <reference types="cypress" />
```

Comment:
Codacy has a fix for the issue: [Expected space or tab after '//' in comment.](https://app.codacy.com/gh/cse110-fa21-group7/cse110-fa21-group7/pullRequest?prid=8427187)
```suggestion
// / <reference types="cypress" />
```